config.py

```diff
@@ -0,0 +1,64 @@
+# For zephyr cron launching only
+

```

Comment:
looks like you committed local config by accident :)

Reply:
Fixes